Library/Formula/glassfish.rb

```diff
@@ -1,26 +1,61 @@
-require "formula"
-
 class Glassfish < Formula
   desc "Java EE application server"
   homepage "https://glassfish.java.net"
   url "http://download.java.net/glassfish/4.1/release/glassfish-4.1.zip"
   sha256 "3edc5fc72b8be241a53eae83c22f274479d70e15bdfba7ba2302da5260f23e9d"
 
+  depends_on :java => "1.7+"
+
+  conflicts_with "payara", :because => "Both install `asadmin` binaries"
+
   def install
     rm_rf Dir["bin/*.bat"]
+    rm "glassfish/config/asenv.bat"
+
+    # Change to point to alternate domains and nodes directory
+    inreplace "glassfish/config/asenv.conf" do |s|
+      s.gsub! %r{AS_DEF_DOMAINS_PATH="../domains"}, "AS_DEF_DOMAINS_PATH=#{var}/payara/domains"
```

Comment:
Why are these `#{var}/payara` instead of `#{var}/glassfish` in the `glassfish` formula, out of interest?